### DISCUSSION TITLE
Can I create workspaces on the fly?

### DISCUSSION THREAD

**Discussion by josephsdavid** [nvim-neorg/neorg#208] [Q&A] [answered]
Title says it all, is there a way I can dynamically create workspaces? My typical workflow is to work in `main folder`, and within that, several `application folders`, and within those, 2 or 3 `feature folders`, and it would be nice to be able to not have to define a ton of directories in my configuration file!

**Comment by mrossinek** [comment, marked as answer]
I understand the question but fail to see the use case. What would the dynamic work space do for you if it is not persistent across your neovim startups?

If you're only looking for a way to jump to certain locations I would rather suggest something like https://github.com/ThePrimeagen/harpoon.
If that is not what you meant, please elaborate on the scenario and what purpose the dynamic workspace serves in your workflow?

**Reply by josephsdavid** [reply]
yeah, now that I have used neorg a bit more, it actually does not make much sense! It is much more simple to have a larger workspace encompassing smaller project, so gtd can help me organize everything!
